### ISSUE TITLE
Subtitles not shown by default

### ISSUE THREAD

**Issue by VasyOk** [videojs/video.js#3894]
## Description
After you add a default subtitle on the list of selected subtitle and "subtitles off". 

![2016-12-27 13-18-43](https://cloud.githubusercontent.com/assets/425040/21497620/4423d338-cc37-11e6-8a41-43cc97920244.png)

## Steps to reproduce
1. Create player
```js
var video = document.createElement('video');
document.body.appendChild(video);

var player = videojs(video, {preload: 'none'});
```
2. Add subtitle and video  
```js
player.addRemoteTextTrack({
    kind: 'subtitles',
    language: 'en',
    label: 'English',
    src: 'link-to-vtt',
    'default': true
});
player.src('link-to-video');
```
3. Load and run player
```js
player.load();
player.play();
```

### Expected
If the default subtitles already added the player should start showing them and select only this subtitle language in menu

### Actual
Both "selected off" and subtitle language are selected in subtitles menu but player does not show subtitles.

### Error output
not errors

#### videojs
5.15.1 (from npm. build with wabpack)
#### videojs-vtt
0.12.1

**Comment by VasyOk** [comment]
for sandbox
```
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="utf-8" />
  <title>Video.js Sandbox</title>

  <!-- Add ES5 shim and sham for IE8 -->
  <script src="../build/temp/ie8/videojs-ie8.js"></script>

  <!-- Load the source files -->
  <link href="../build/temp/video-js.css" rel="stylesheet" type="text/css">
  <script src="../build/temp/video.js"></script>

  <!-- Set the location of the flash SWF -->
  <script>
    videojs.options.flash.swf = '../build/temp/video-js.swf';
  </script>

</head>
<body>
<div style="background-color:#eee; border: 1px solid #777; padding: 10px; margin-bottom: 20px; font-size: .8em; line-height: 1.5em; font-family: Verdana, sans-serif;">
  <p>You can use /sandbox/ for writing and testing your own code. Nothing in /sandbox/ will get checked into the repo, except files that end in .example (so don't edit or add those files). To get started make a copy of index.html.example and rename it to index.html.</p>
  <pre>cp sandbox/index.html.example sandbox/index.html</pre>
  <pre>grunt watch</pre>
  <pre>grunt connect</pre>
  <pre>open http://localhost:9999/sandbox/index.html</pre>
</div>

<a href="#" id="pl-btn">run</a>

<script>
  var video = document.createElement('video');
  video.controls = true;
  video.setAttribute('width', '640');
  video.setAttribute('height', '264');
  video.classList.add('video-js');
  video.classList.add('vjs-default-skin');
  document.body.appendChild(video);

  var player = videojs(video, {preload: 'none'});
  player.load();

  var btn = document.getElementById('pl-btn');
  btn.addEventListener('click', function (e) {
    e.preventDefault();

    player.addRemoteTextTrack({
      kind: 'subtitles',
      language: 'en',
      label: 'English',
      src: '../docs/examples/shared/example-captions.vtt',
      'default': true
    }, true);
    player.src('http://vjs.zencdn.net/v/oceans.mp4');

    player.load();
    player.play();
  });
</script>
</body>
</html>
```

by clicking on "run"

**Comment by gkatsev** [comment]
What browser are you running this on? Based on the symptoms it looks like it's Safari. Safari has a [bug where it always show subtitles by default](https://bugs.webkit.org/show_bug.cgi?id=147951). Both of the items showing up as selected *is* a videojs bug, though.

**Comment by BrandonOCasey** [comment]
There is a fix for this coming in #4217 & #4216

**Comment by BrandonOCasey** [comment]
#4217 & #4216 are merged this should be fixed in the next release of video.js 5 & video.js 6
